### PR TITLE
Add option to next.config docs

### PR DESCRIPTION
**Pull request by molebox**
Adds docs for PPR to `next.config` section

### CODE REVIEW

File: docs/02-app/02-api-reference/05-next-config-js/partial-prerendering.mdx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: ppr (experimental)
+description: Enable Partial Prerendering (PPR) in Next.js 14.
+---
+
+{/* The content of this doc is shared between the app and pages router. You can use the `<PagesOnly>Content</PagesOnly>` component to add content that is specific to the Pages Router. Any shared content should not be wrapped in a component. */}
```
Comment (delbaoliveira) [inline docs/02-app/02-api-reference/05-next-config-js/partial-prerendering.mdx:6]: You can delete this
```suggestion
```

File: docs/02-app/02-api-reference/05-next-config-js/partial-prerendering.mdx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+<AppOnly>
```
Comment (delbaoliveira) [inline docs/02-app/02-api-reference/05-next-config-js/partial-prerendering.mdx:8]:
```suggestion
```

File: docs/02-app/02-api-reference/05-next-config-js/partial-prerendering.mdx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+const nextConfig = {
+  experimental: {
+    ppr: true,
+  },
+};
+ 
+module.exports = nextConfig;
+```
+
+</AppOnly>
```
Comment (delbaoliveira) [inline docs/02-app/02-api-reference/05-next-config-js/partial-prerendering.mdx:25]:
```suggestion
```

File: docs/02-app/02-api-reference/05-next-config-js/partial-prerendering.mdx [hunk 4]
```diff
@@ -0,0 +1,25 @@
+---
+title: ppr (experimental)
+description: Enable Partial Prerendering (PPR) in Next.js 14.
+---
+
+{/* The content of this doc is shared between the app and pages router. You can use the `<PagesOnly>Content</PagesOnly>` component to add content that is specific to the Pages Router. Any shared content should not be wrapped in a component. */}
+
+<AppOnly>
+
+Experimental support for Partial Prerendering (PPR). PPR is available in Next.js 14 and allows static portions of a page to be pre-generated and served from the cache with dynamic holes streamed in a single HTTP request.
```
Comment (delbaoliveira) [inline docs/02-app/02-api-reference/05-next-config-js/partial-prerendering.mdx:10]:
```suggestion
Experimental support for Partial Prerendering (PPR). PPR is available in Next.js 14 and allows static portions of a route to be pre-generated and served from the cache with dynamic holes streamed in a single HTTP request.
```

**Review by delbaoliveira — CHANGES_REQUESTED** [review]

**Review by delbaoliveira — DISMISSED** [review]

File: docs/02-app/02-api-reference/05-next-config-js/partial-prerendering.mdx [hunk 5]
```diff
@@ -0,0 +1,21 @@
+---
+title: ppr (experimental)
```
Comment (leerob) [inline docs/02-app/02-api-reference/05-next-config-js/partial-prerendering.mdx:2]:
```suggestion
title: Partial Prerendering
```

File: docs/02-app/02-api-reference/05-next-config-js/partial-prerendering.mdx [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+description: Enable Partial Prerendering (PPR) in Next.js 14.
```
Comment (leerob) [inline docs/02-app/02-api-reference/05-next-config-js/partial-prerendering.mdx:3]:
```suggestion
description: Learn how to enable Partial Prerendering (experimental) in Next.js 14.
```

File: docs/02-app/02-api-reference/05-next-config-js/partial-prerendering.mdx [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+---
+
+Experimental support for Partial Prerendering (PPR). PPR is available in Next.js 14 and allows static portions of a route to be pre-generated and served from the cache with dynamic holes streamed in a single HTTP request.
```
Comment (leerob) [inline docs/02-app/02-api-reference/05-next-config-js/partial-prerendering.mdx:6]:
```suggestion
Partial Prerendering allows static portions of a route to be prerendered and served from the cache with dynamic holes streamed in, all in a single HTTP request.
```

File: docs/02-app/02-api-reference/05-next-config-js/partial-prerendering.mdx [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+
+You can enable PPR in your Next.js project by setting the experimental `ppr` flag:
```
Comment (leerob) [inline docs/02-app/02-api-reference/05-next-config-js/partial-prerendering.mdx:8]:
```suggestion
You can enable Partial Prerendering by setting the experimental `ppr` flag:
```

File: docs/02-app/02-api-reference/05-next-config-js/partial-prerendering.mdx [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+const nextConfig = {
+  experimental: {
+    ppr: true,
+  },
+}
+
+module.exports = nextConfig
+```
+
+Learn more about PPR in the [Next.js learn course](/learn/dashboard-app/partial-prerendering).
```
Comment (leerob) [inline docs/02-app/02-api-reference/05-next-config-js/partial-prerendering.mdx:21]:
```suggestion
Learn more about Partial Prerendering in the [Next.js Learn course](/learn/dashboard-app/partial-prerendering).
```

**Comment by leerob** [comment]
Prefer "Partial Prerendering" over PPR for awhile. Maybe we'll get to an acronym some day.

**Review by delbaoliveira — APPROVED** [review]

**Comment by leerob** [comment]
Going to fast follow with this 🙏 Will come back.

**Pull request closed without merging**